client/src/components/settings/toggle-radio-setting.tsx

```diff
@@ -41,26 +41,43 @@ export default function ToggleRadioSetting({
           {explain ? <p id={`desc${flagName}`}>{explain}</p> : null}
         </div>
         <div className='toggle-radio-group'>
-          <label htmlFor={firstRadioId} data-checked={flag}>
+          <label
+            htmlFor={firstRadioId}
+            className='toggle-radio-label'
+            data-checked={flag}
```

Comment:
I don't think we need the `data-checked` attribute on the `label`.

Reply:
Correct. data-checked could be removed. It works fine.